docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md

```diff
@@ -84,9 +75,24 @@ To submit an **update** for an already published plugin:
      - Select **Multiple** if you'd like to submit separate plugin archives for each architecture.
        This can lead to faster downloads since users can select the specific architecture they want to install the plugin on.
    - **URL:** A URL that points to a ZIP archive of your packaged plugin.
-   - **Source Code URL:** A URL that points to a public Git repository or ZIP archive of your complete plugin source code. See [examples](#what-source-code-url-formats-are-supported).
+   - **Source code URL:** A URL that points to a public Git repository or ZIP archive of your complete plugin source code. See [examples](#what-source-code-url-formats-are-supported).
    - **MD5:** The MD5 hash of the plugin specified by the **URL**.
 1. Click **Submit**.
+    All plugin submissions, new or updates, go through the same automated and rigorous manual review process. Because we may have a test environment already setup for an existing plugin, plugin update reviews may go faster.
+
+## Deprecate a plugin
```

Comment:
On your question. Given the title of this page is _Publish or update a plugin_ it could make sense to put it in a separate page. However, it's still related to deprecate a plugin so could stay as well. For now, lets keep it here would be my suggestions

Reply:
I have at least temporarily moved deprecation to a separate page AND kept it on this page as an FAQ. I'm open to other ideas though if other reviewers feel strongly about one way or the other.

Reply:
@sympatheticmoose - Shoot, I forgot to check in the commit (deprecation.md is now added). I'll leave it up to you to decide if/where it goes up. Could you clarify if you want to move it to create-a-plugin/extend-a-plugin or leave it out? I'm not 100% sure based on your comment above.

Reply:
I prefer a section and left a suggestion on the content for it here https://github.com/grafana/plugin-tools/pull/462#discussion_r1366816623.

Note the empty page is still showing for me
<img width="295" alt="Screenshot 2023-10-23 at 14 44 22" src="https://github.com/grafana/plugin-tools/assets/19860021/1854eea6-c497-40d5-b5bf-5deced4594b7">

Reply:
Should be resolved by https://github.com/grafana/plugin-tools/pull/462/commits/bed1e6c29eb6f316177a96fe9d32ed925d20859d

Reply:
<img width="877" alt="Screenshot 2023-10-26 at 20 45 49" src="https://github.com/grafana/plugin-tools/assets/19860021/475f5775-9fe9-4577-b8b0-c0612a1e1576">
still there